spec/spec_helper.rb

```diff
@@ -5,7 +5,9 @@
 require 'simplecov'
 require 'erb'
 require 'yaml'
+require 'active_support'
```

Comment:
This is to fix Timecop issue. (Actually the dependency issue between `active_support` and `active_support/core_ext`.
`active_support` has to be required prior to `active_support/core_ext`

```
  5) Integration Shinq::Client.enqueue,dequeue with scheduled_at on table when scheduled_at is specified can not dequeue job immediately
     Failure/Error: Timecop.freeze

     NameError:
       uninitialized constant ActiveSupport::IsolatedExecutionState

             ::ActiveSupport::IsolatedExecutionState[:time_zone] || zone_default
                            ^^^^^^^^^^^^^^^^^^^^^^^^
     # ./vendor/bundle/ruby/3.1.0/gems/activesupport-7.0.4.2/lib/active_support/core_ext/time/zones.rb:15:in `zone'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/time_stack_item.rb:137:in `time_klass'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/time_stack_item.rb:111:in `parse_time'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/time_stack_item.rb:12:in `initialize'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/timecop.rb:192:in `new'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/timecop.rb:192:in `travel'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/timecop.rb:131:in `send_travel'
     # ./vendor/bundle/ruby/3.1.0/gems/timecop-0.9.6/lib/timecop/timecop.rb:52:in `freeze'
     # ./spec/integration_spec.rb:26:in `block (3 levels) in <top (required)>'
```